CVs/input4MIPs_source_id.json

```diff
@@ -1,4 +1,20 @@
 {
+    "CEDS-CMIP-2024-07-08":{
+        "contact":"Rachel Hoesly (rachel.hoesly@pnnl.gov); Steve Smith (ssmith@pnnl.gov)",
+        "further_info_url":"https://www.pnnl.gov/projects/ceds",
+        "institution_id":"PNNL-JGCRI",
+        "license_id":"CC BY 4.0",
+        "mip_era":"CMIP6Plus",
+        "source_version":"2024-07-08"
+    },
+    "CEDS-CMIP-2024-07-08-supplemental":{
+        "contact":"Rachel Hoesly (rachel.hoesly@pnnl.gov); Steve Smith (ssmith@pnnl.gov)",
+        "further_info_url":"https://www.pnnl.gov/projects/ceds",
+        "institution_id":"PNNL-JGCRI",
+        "license_id":"CC BY 4.0",
+        "mip_era":"CMIP6Plus",
+        "source_version":"2024-07-08"
+    },
```

Comment:
@ssmithClimate @hahsan1 can you please confirm these are the entries you want.

Reply:
Looks good to me.